plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesScanner.java

```diff
@@ -189,6 +177,8 @@ private void copyAffectedFiles(final Report report, final FilePath buildFolder)
             permittedSourceDirectories.add(workspace.getRemote());
             new AffectedFilesResolver().copyAffectedFilesToBuildFolder(
                     report, workspace, permittedSourceDirectories, buildFolder);
+        } else {
```

Comment:
Check your codingstyle settings.

```suggestion
        } 
        else {
```